lambo

```diff
@@ -117,12 +117,21 @@ fi
 
 # Update .env to point to this database with `root` username and blank pw,
 # like Mac MySQL defaults, and appropriate domain
-sed -i '' "/DB_DATABASE/s/homestead/$PROJECTNAME/" .env
-sed -i '' '/DB_USERNAME/s/homestead/root/' .env
-sed -i '' '/DB_PASSWORD/s/secret//' .env
-sed -i '' "/APP_URL/s/localhost/$PROJECTNAME\.$TLD/" .env
-
-open "http://$PROJECTNAME.$TLD"
+if [ "$(uname)" == "Darwin" ]; then
+    sed -i '' "/DB_DATABASE/s/homestead/$PROJECTNAME/" .env
```

Comment:
Maybe we could do it something like this? http://www.cyberciti.biz/faq/bash-for-loop-array/

It looks like the strings are the same, right? If so, then we can store them in an array and just let the command be different each time